### PR TITLE
openstack: document common pitfalls and OpenStack tenant grants

### PR DESCRIPTION
**Pull request by dachary**
Signed-off-by: Loic Dachary <loic@dachary.org>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -330,10 +330,15 @@ using OpenStack virtual machines.
...
-available (for instance https://www.ovh.com/fr/cloud/), some jobs
-won't run because they expect volumes attached to each instance.
+An OpenStack tenant with access to the nova and cinder API. If the
+cinder API is not available, some jobs won't run because they expect
+volumes attached to each instance.
+
+Apply for an OpenStack account
+------------------------------
+
+* Send a mail to `Loic Dachary <mailto:loic@dachary.org>`_
```
Comment (zmc) [inline README.rst]: Some more info about these accounts which are open for application would be good - it's the first I've heard of it.

File: README.rst [hunk 2]
```diff
@@ -400,8 +405,11 @@ Usage
     $ openstack keypair create myself > myself.pem
     $ chmod 600 myself.pem
 
-* Run the dummy suite (it does nothing useful but shows all works as
-  expected)::
+* Run the dummy suite. It does nothing useful but shows all works as
+  expected. Note that it can take a long time to run (from a few
```
Comment (zmc) [inline README.rst]: Maybe "the first time it is run, it can take a long time"

File: README.rst (hunk 1 above)
Comment (dachary) [inline README.rst]: This is a temporary measure, until a more permanent solution is found. I forwarded you the ceph-devel mail that explains it with a bit more details (title "Enabling contributors to run teuthology suites")

File: README.rst (hunk 2 above)
Comment (dachary) [inline README.rst]: Thanks, much better. I so wish I could write proper english ...

**Comment by dachary** [comment]
Modified and repushed ! Thanks :-)

**Pull request merged**